openedge-plugin/src/main/java/org/sonar/plugins/openedge/sensor/OpenEdgeCodeColorizer.java

```diff
@@ -104,11 +104,18 @@ private void highlightFile(SensorContext context, RefactorSession session, Input
       }
 
       if (textType != null) {
-        TextPointer start = file.newPointer(tok.getLine(), tok.getColumn() - 1);
-        int maxChar = file.selectLine(tok.getEndLine()).end().lineOffset();
-        TextPointer end = file.newPointer(tok.getEndLine(),
-            maxChar < tok.getEndColumn() ? maxChar - 1 : tok.getEndColumn());
-        highlighting.highlight(file.newRange(start, end), textType);
+        try {
+          TextPointer start = file.newPointer(tok.getLine(), tok.getColumn() - 1);
+          int maxChar = file.selectLine(tok.getEndLine()).end().lineOffset();
+          TextPointer end = file.newPointer(tok.getEndLine(),
+              maxChar < tok.getEndColumn() ? maxChar - 1 : tok.getEndColumn());
+
+          highlighting.highlight(file.newRange(start, end), textType);
+        } catch (IllegalArgumentException caught) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1166)